.travis.yml

```diff
@@ -5,11 +5,13 @@ python:
 - '3.4'
 - '3.5'
 install:
-- pip install -r requirements.txt
```

Comment:
Changes to `.travis.yml` should be a separate PR; from what I can tell this isn't related to the DRY refactor in any way.